deps/rabbit/test/clustering_management_SUITE.erl

```diff
@@ -752,11 +752,19 @@ equal(Status0, Status1) ->
     Status0 == Status1.
 
 cluster_status(Node) ->
+    {AllNodes, DiscNodes, RunningNodes} =
+    try
+        erpc:call(Node, rabbit_db_cluster, current_or_last_status, [])
+    catch
+        error:{exception, undef,
+               [{rabbit_db_cluster, current_or_last_status, _, _} | _]} ->
+            erpc:call(Node, rabbit_mnesia, cluster_status, [status])
+    end,
     {rpc:call(Node, rabbit_nodes, list_members, []),
      rpc:call(Node, rabbit_nodes, list_running, []),
-     rpc:call(Node, rabbit_mnesia, cluster_nodes, [all]),
```

Comment:
This willfully calls `rabbit_mnesia` so that when a new metadata store is used, we can compare `mnesia` (which is always running at least until 4.x) with the status of the currently enabled store (returned by `rabbit_nodes` above, could be `mnesia` or `khepri` in the future). That said, if we decide to disable `mnesia` as soon as we enable another store, the test as it was needs to be updated.

Reply:
As discussed outside of this pull request with Diana, Mnesia and Khepri will not be used at the same time. Therefore, we can start to reduce the use of `rabbit_mnesia`.

Reply:
Yes, that can now be removed